@hmlnarik 

Update #2: Actually taking back. Unzip is in both modules `sso` and `keycloak/openshift/clients`, so it needs to be on both places.

Another problem is (since JDK isn't installed yet), the `$JBOSS_HOME` variable isn't defined yet, leading to:
```
2019-10-24 12:02:46,396 cekit        INFO     Docker: + SOURCES_DIR=/tmp/artifacts
2019-10-24 12:02:46,396 cekit        INFO     Docker: + unzip -o /tmp/artifacts/keycloak-server-overlay.zip -d
2019-10-24 12:02:46,396 cekit        INFO     Docker: error:  must specify directory to which to extract with -d option
2019-10-24 12:02:46,530 cekit        ERROR    You can look inside the failed image by running 'docker run --rm -ti 744129886cb5
 bash'
2019-10-24 12:02:46,531 cekit        ERROR    ('Image build failed, see logs above.', CekitError("Image build failed: 'The command 'bash -x /tmp/scripts/sso/install' returned a non-zero code: 10'",))
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/cekit/cli.py", line 256, in run
    builder.build()
  File "/usr/lib/python2.7/site-packages/cekit/builders/docker_builder.py", line 98, in build
    raise CekitError(msg, ex)
CekitError: ('Image build failed, see logs above.', CekitError("Image build failed: 'The command 'bash -x /tmp/scripts/sso/install' returned a non-zero code: 10'",))
```
To workaround this, I would suggest to add JBOSS_HOME var definition into the `keycloak/openshift/clients` module:
https://github.com/jboss-container-images/redhat-sso-7-image/blob/sso73-dev/modules/keycloak/openshift/clients/install#L8

something like:

`if [! -v $JBOSS_HOME ]; then
   export JBOSS_HOME="/opt/eap"
fi
`
yet before the `unzip` call in the same `keycloak/openshift/clients` module is called.

The above setting comes from current RH-SSO 7.3 image:
```
$ docker run -it 102b1151daf5 /bin/bash
[jboss@b8e42e727446 ~]$ echo $JBOSS_HOME
/opt/eap
```

--- a/image.yaml
+++ b/image.yaml
@@ -3,7 +3,7 @@ schema_version: 1
 name: "redhat-sso-7/sso73"
 description: "Red Hat Single Sign-On 7.3 container image"
 version: "7.3.4.GA"
-from: "jboss-eap-7/eap72:7.2.4-1"
+from: "rhel7:7-released"
 labels:
     - name: "com.redhat.component"
       value: "redhat-sso-7-sso73-container"
@@ -20,10 +20,22 @@ envs:
       value: "7.3.4.GA"
     - name: "PRODUCT_VERSION"
       value: "7.3.4.GA"
+
 modules:
       repositories:
           - path: modules
+          - name: cct_module
+            git:
+                url: https://github.com/jboss-openshift/cct_module.git
+                ref: 0.37.1
+          - name: jboss-eap-7-image
+            git:
+                url: https://github.com/jboss-container-images/jboss-eap-7-image.git
+                ref: EAP_724_CR2
       install:
+          - name: jboss.container.openjdk.jdk
+            version: '8'
+          - name: eap-7.2-latest
           - name: sso
             version: '1.0'
           - name: keycloak.openshift.clients

